org.freedesktop.Geoclue.Data.xml

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.freedesktop.Geoclue.Data">
```

Comment:
This would be your own api? If so, why call it geoclue? Could be some hybrisprovider specific?

Reply:
GPS is part of the geoclue. cold start is part of the GPS. Accordingly, this should relate directly to geoclue.

Reply:
You seem to be anyway introducing a new api which is hybris specific. It's not part of geoclue, just an extra interface on a geoclue provider.

Furthermore you've multiple times suggested to use the common options api way which doesn't need any new interface, yet no rationale given why it wouldn't be good.

Reply:
Please check the updated code, I added a new interface com.omp.gps.service